packages/fiori/src/SideNavigationGroupTemplate.tsx

```diff
@@ -45,6 +44,7 @@ function TreeItemTemplate(this: SideNavigationGroup) {
 				<ul id={this._groupId}
 					class="ui5-sn-item-ul"
 					role="group"
+					aria-label={this.text}
```

Comment:
Now when you focus a group, its name is announced twice. For example "Group 2 Group 2"